static/src/javascripts/projects/common/modules/commercial/acquisitions-copy.js

```diff
@@ -39,10 +39,8 @@ export const usLocalisedFlag = {
 export const liveblog = (
     membershipUrl: string,
     contributionsUrl: string,
-    useSupportDomain: boolean = false
+    supportP2: string
```

Comment:
Is this "support paragraph 2"?  It is probably obvious to most people as is, but I read it as a P(riority)2 support incident.

Reply:
Ha, yes. It's really just a component of the ask paragraph, the bit with the CTA links. The convention before was to use something called `controlP2` so I borrowed from that. Can change it to something like `ctaLinkSentence` or something, if it's upsetting as-is.